javers-core/src/main/java/org/javers/core/graph/ObjectNode.java

```diff
@@ -75,7 +78,11 @@ public GlobalId getReference(Property property){
 
     public Object getPropertyValue(Property property) {
         Validate.argumentIsNotNull(property);
-        return cdo.getPropertyValue(property);
+        try {
+            return cdo.getPropertyValue(property);
+        } catch (JaversGetterException e){
```

Comment:
no właśnie nie ma i nie powinno być, użytkownika nie interesują niskopoziomowe wyjątki w rodzaju IllegalArgumentException z mrocznym komunikatem:
`java.lang.IllegalArgumentException: Can not set java.lang.String field org.javers.core.cases.TypeMismatchExceptionClasses$B0.b0 to org.javers.core.cases.TypeMismatchExceptionClasses$B1`
rzucane z java.lang.reflect tylko informacja, co ma skopane w mapowaniu obiektów domenowych  #informationHiding #encapsulation